### PR TITLE
Project-Movie-ClickaChill (Tara)

### PR DESCRIPTION
**Pull request by trangtara**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

**Review by emeliesv — CHANGES_REQUESTED** [review]
🤩 🤩 Very nice work Tara! You have made a beautiful page with awesome animations! 🤩 🤩 
 
I noticed that you, just like me, aren't consistent with semicolons. On some places you use them, on others you don't. Perhaps you have the same issue as I had with the auto-formatter using semicolons as a default and you yourself do not? 😊  

I ran your page in lighthouse and it looks good for the most part! You could work a bit more on the accessability in desktop and see if you could do anything about the imagesizes in mobile maybe? 🤔 I tried running it in wave as well, but that didn't work 😅 

I also noticed that your beautiful error-message isn't displayed when the site is deployed on netlify, I only got netlifys standard 404. I don't know why this happened, my page has the same issue (so something to do with netlify?). I also noticed that you dropdown has a white background in chrome and edge. I posted a screenshot of that in the comments.

Look over your usage of semicolons and remove empty lines and your good to go!
😍 😍 😍

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,48 @@
...
+          <Cast />
+        </Route>
+        <Route path="/profile/cast/:castId">
+          <MovieProjectsByCast />
+        </Route>
+        <Route path="/movies/:movieId/crew" exact>
+          <Crew />
+        </Route>
+        <Route path="/profile/crew/:crewId">
+          <MovieProjectsByCrew />
```
Comment (emeliesv) [inline code/src/App.js:37]: Perhaps Cast and Crew could be the same component and you just send in different props? If you'd like to clean up the code a bit 😉 🥰

File: code/src/style/movieList.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+.select-container {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 30px;
+  margin-top: 30px;
+  width: 100vw;
+}
+
+.select-input {
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/style/movieList.css:27]: Thanks for spotting this Emelie.

File: code/src/App.js (hunk 1 above)
Comment (trangtara) [reply code/src/App.js:37]: Yeah, i guess so. They are kind of repetitive.

File: code/src/pages/Crew.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+import '../style/displayStyling.css'
+import '../style/linkStyling.css'
+
+const Crew = () => {
+  const { movieId } = useParams();
+  const [crewList, setCrewList] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/movie/${movieId}/credits?api_key=fd00bf32c04d62eccaf294e2d6aa9fa6`)
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/pages/Crew.js:12]: Agree!

File: code/src/constant.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export const API_KEY = "fd00bf32c04d62eccaf294e2d6aa9fa6";
+
+export const MOVIES_API = "https://api.themoviedb.org/3/movie/popular?api_key=fd00bf32c04d62eccaf294e2d6aa9fa6&language=en-US&page=1";
+
+export const upComingMovies = "https://api.themoviedb.org/3/movie/upcoming?api_key=fd00bf32c04d62eccaf294e2d6aa9fa6&language=en-US&page=1";
+
+export const nowPlayMovies = "https://api.themoviedb.org/3/movie/now_playing?api_key=fd00bf32c04d62eccaf294e2d6aa9fa6&language=en-US&page=1";
+
+export const topRatedMovies = "https://api.themoviedb.org/3/movie/top_rated?api_key=fd00bf32c04d62eccaf294e2d6aa9fa6&language=en-US&page=1"
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/constant.js:9]: thanks!

File: code/src/App.js [hunk 5]
```diff
@@ -1,9 +1,48 @@
...
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <Switch>
+        <Route path="/" exact>
+          <MovieList />
+        </Route>
+        <Route path="/movies" exact>
+          <Redirect to="/" />
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/App.js:22]: this means even when the path just has "/", the page will still be directed to the Movie, which in this case is the home page.

File: code/src/pages/MovieList.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import '../style/movieList.css'
+import { MOVIES_API } from '../constant'
+
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/pages/MovieList.js:5]: Hahaa, yes!!! I am too generous with space!!haha

File: code/src/pages/PageNotFound.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import '../style/pageNotFound.css'
+
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/pages/PageNotFound.js:4]: I leave a line between external import (from React) and internal import (from css or components or page file) just to distinguish them. No right no wrong, I just remembered Maks recommended that.

File: code/src/pages/Crew.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        setCrewList(json.crew)
+      });
+  }, [movieId]);
+
+  return ( 
+    <div className="display-card-container">
+      {crewList.map((crew) => (
+        <Link 
+          className="links" 
+          to={`/profile/crew/${crew.id}`}>
```
Comment (EmOlsz) [inline code/src/pages/Crew.js:24]: Missing `key` attribute

Comment (trangtara) [reply code/src/pages/Crew.js:25]: Key={crew.credit_id} added.

**Pull request closed without merging**